javascript/javascript3/week1/giphy_App/giphy.js

```diff
@@ -0,0 +1,45 @@
+//select the to inputs 
+const page = document.querySelector('body')
```

Comment:
Try and limit the use of global variables/constants. See if you can order everything into small functions